MNE/rtClient/rtdataclient.cpp

```diff
@@ -290,7 +291,8 @@ FiffInfo::SPtr RtDataClient::readInfo()
                                 p_pFiffInfo->comps[countComp].data->col_names = FiffStream::split_name_list(col_names);
                             break;
                         case FIFF_MNE_CTF_COMP_DATA:
-                                p_pFiffInfo->comps[countComp].data->data = t_pTag->toNamedMatrix().cast<double>();
+                            //ToDo: Test; Float Matrix
+                            p_pFiffInfo->comps[countComp].data->data = t_pTag->toNamedMatrix().cast<double>();
```

Comment:
Was this tested using the FiffSimulator and a fiff file with available compensators (i.e. a ctf file)?